.github/workflows/poetry.yml

```diff
@@ -37,9 +37,9 @@ jobs:
         run: |
           poetry run pip install coverage[toml]
         if: startsWith(matrix.os, 'ubuntu')
-      - name: Install examples dependencies
+      - name: Install examples and test dependencies
         run: |
-          poetry install --only main --extras examples
+          poetry install --only main --extras "examples test"
```

Comment:
Do we really want this? The examples should not depend on the test dependencies.

Reply:
Ah good catch! I added it to the wrong command. I've created #185 to fix this... Thanks!